service/microceph.go

```diff
@@ -16,6 +15,7 @@ import (
 	"github.com/canonical/microcluster/microcluster"
 
 	"github.com/canonical/microcloud/microcloud/api/types"
+	cloudCli "github.com/canonical/microcloud/microcloud/client"
```

Comment:
Can we call this `microcloudClient` or `cloudClient` to be consistent with the `cephClient` above?

Reply:
This one is still open, rest looks good.